tools/runners/UhdmYosys.py

```diff
@@ -32,7 +32,7 @@ def prepare_run_cb(self, tmp_dir, params):
 
         # generate yosys script
         with open(yosys_scr, "w") as f:
-            f.write("plugin -i uhdm\n")
+            f.write("plugin -i systemverilog\n")
             f.write("read_uhdm slpp_all/surelog.uhdm\n")
```

Comment:
Would there be any benefit in also testing `read_systemverilog` directly? Or maybe doing this instead of `read_uhdm`?

Reply:
I think we should use``read_systemverilog`` directly, as it is probably easier command for users. I've adapted ``UhdmYosys`` runner to use it.